spec/request_handler_spec.rb

```diff
@@ -71,17 +71,36 @@
         end
       end
 
-      context "when request contains special characters" do
-        let(:valid_attributes) { attributes.merge(address_line2: "àéïø") }
+      describe "authorized special characters" do
+        context "when a field contains 32 decomposed unicode characters" do
```

Comment:
It would be useful if you could leave a comment explaining what is "decomposed unicode characters". It will be a good refresher after some time as well something useful for new developers. Same for compose special characters.